features/gherkin_parse_failure.feature

```diff
@@ -16,5 +16,30 @@ Feature: Gherkin parse failure
     Then it fails
     And the error output contains the text:
       """
-      Parse error in 'features/a.feature'
+      Parse error in "features/a.feature" (4:5)
       """
+
+  Scenario: All parseable sources and all parse errors are emitted
+    Given a file named "features/a.feature" with:
+      """
+      Parse Error
+      """
+    Given a file named "features/b.feature" with:
+      """
+      Feature: a feature name
+        Scenario: a scenario name
+          Given a step
+      """
+    Given a file named "features/c.feature" with:
+      """
+      Parse Error
+      """
+    When I run cucumber-js with `--format message`
+    Then it fails
+    And the output contains these messages:
```

Comment:
I would just rename the step to make it ckear we check types and quantity of messages rather than the messages

Reply:
Yep good point, updated now.